### PR TITLE
Task/update integration tests

### PR DESCRIPTION
**Pull request by gbudau**
This PR adds tests for known issues, which should pass after we fix them.

### CODE REVIEW

File: srcs/app/test/integration/webserv_test.py [hunk 1]
```diff
@@ -125,8 +125,37 @@ def test_url_path_traversal():
...
     filename = '../../../../../../../../../../../../../../tmp/webserv/' + random_filename
-    mime_type = 'application/octet-stream'
-    files = {'file': (random_filename, 'random text\n')}
+    mime_type = 'text/plain'
+    files = {'file': (filename, 'random text\n', mime_type)}
     response = requests.post(url, files=files)
     assert not os.path.exists('/tmp/webserv/' + random_filename)
     assert response.status_code == 400
+
+@pytest.mark.skip(reason="known issue")
```
Comment (gbudau) [inline srcs/app/test/integration/webserv_test.py:134]: To enable skipped tests, delete the line starting with `@pytest.mark.skip`

**Review by Damrod — APPROVED** [review]
Looks great, thank you!
(I would like to meet up some day to look into how to implement an integration test on the topic of the non blocking cgi...)

**Comment by gbudau** [comment]
@Damrod, thank you!

For the integration tests of non-blocking cgi, I've made draft #122. Let me know your thoughts.

**Review by vdedios — APPROVED** [review]
LGTM!!

**Pull request merged**